apps/o2-blocks/src/github-issue-template-button/editor.scss

```diff
@@ -0,0 +1,9 @@
+.wp-block {
```

Comment:
We probably need a block-specific class here to avoid styling other blocks with this sheet, i.e. `wp-block-github-issue-template-button`.

See for ex.: https://github.com/Automattic/wp-calypso/blob/ced5155e67fdd5234f76e411f5bdb74e587871e4/apps/o2-blocks/src/project-status/editor.scss#L1.

Reply:
You're totally right, this will affect other blocks as Jon mentioned. Apologies and thanks for pointing that out!